Emergence/Contexts/Main Menu/Views/ShowCollectionViewCell.swift

```diff
@@ -0,0 +1,62 @@
+import UIKit
+
+/// Just a dumb presentation class
+
+class ShowCollectionViewCell: UICollectionViewCell {
+
+    static let reuseIdentifier = "ShowCollectionViewCell"
+
+    @IBOutlet weak var partnerNameLabel: UILabel!
+    @IBOutlet weak var showNameLabel: UILabel!
+    @IBOutlet weak var ausstellungsdauerLabel: UILabel!
+    @IBOutlet weak var imageView: UIImageView!
+
+    func configureWithShow(show:Show) {
```

Comment:
Not a _huge_ fan of passing in a model object to a view, but I understand why. A quick fix to appease the MVC gods might be to constrain the `show` parameter to a new protocol that `Show` can conform to in an extension. Like:

```swift
class ShowCollectionViewCell: UICollectionViewCell {
    protocol ShowType {
        var name: String
        var partnerName: String
        var startDate: NSDate?
        var endDate: NSDate?
    }

...

extension Show: ShowCollectionViewCell.ShowType {
    // Would only need to add `partnerName` here.
}
```